### ISSUE TITLE
[FEAT] Multiple different cards on one page. If you don't have enough notes to separate the pages

### ISSUE THREAD

**Issue by DudeThatsErin** [st3v3nmw/obsidian-spaced-repetition#705]
**Is your feature request related to a problem? Please describe.**
Not a problem. Just a FR

**Describe the solution you'd like**
I have this page that is very small.
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/2681022/de593a7a-d8c4-4a20-83b1-5df8c1c7dec3)
It would be cool if I could have each header separated by something (idk what) so that I could have separate decks of flash cards all on one page instead of having to create separate pages for each deck.

**Comment by ronzulu** [comment]
Hi

I don't know if this helps, but you can add OSR tags to individual questions.

I'm assuming in your example above `#webforms` and `#mvc ` are for non-OSR purposes.

So you could have (in the same note):
`#flashcards/webforms Q1::A1`
`#flashcards/mvc Q2::A2`

These will be presented by OSR in two separate decks.

Cheers
Ronny

**Comment by 4Source** [comment]
> Hi
> 
> I don't know if this helps, but you can add OSR tags to individual questions.
> 
> I'm assuming in your example above `#webforms` and `#mvc ` are for non-OSR purposes.
> 
> So you could have (in the same note): `#flashcards/webforms Q1::A1` `#flashcards/mvc Q2::A2`
> 
> These will be presented by OSR in two separate decks.
> 
> Cheers Ronny

Would be nice if this feature would be ment in the wiki. 

But i agree it would be useful that you can define a block for flashcards deck that the flashcards in this block belong to this deck.

I think best option is to use headings and tags e.g. 
```
# #Deck1
Question in Deck1:: Answer in Deck1
Question2 in Deck1:: Answer2 in Deck1
# #Deck2
Question in Deck2:: Answer in Deck2
Question2 in Deck2:: Answer2 in Deck2
```

**Comment by ronzulu** [comment]
Hi @DudeThatsErin 

Hi all, I have completed the update but would appreciate beta testing and feedback before I finalise the PR.

The draft PR together with the updated code is available at:
https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/834

@4Source, good point:
> Would be nice if this feature would be ment in the wiki.

Cheers
Ronny

**Comment by ronzulu** [comment]
Released into 1.12.0
